examples/example-next-static-export/src/env.test.ts

```diff
@@ -0,0 +1,33 @@
+import { loadEnvConfig } from "@next/env";
+import { afterEach, beforeEach, expect, test, vi } from "vitest";
+
+const ENV_VARS = [
+  "NEXT_PUBLIC_FOUNDRY_API_URL",
+  "NEXT_PUBLIC_FOUNDRY_CLIENT_ID",
+  "NEXT_PUBLIC_FOUNDRY_REDIRECT_URL",
+];
+
+beforeEach(() => {
+  vi.stubEnv("NODE_ENV", "production");
```

Comment:
Stubbed for `@next/env` which doesn't allow specifying mode as an input:
https://github.com/vercel/next.js/blob/adc94b49d47cd263140ba6523bc2db2e6d09e0d2/packages/next-env/index.ts#L110-L142